src/components/common/Navigation.tsx

```diff
@@ -13,7 +13,7 @@ const Navigation = () => {
         <IconSettings />
         <Desc>설정</Desc>
       </LinkItem>
-      <LinkItem href="/settings">
+      <LinkItem href="/setting">
```

Comment:
이거 왜 `setting`으로 바꾸신 건가요?

Reply:
```suggestion
      <LinkItem href="/setting">
```

전부 settings 이길래 수정하려다가 만것같습니다